src/database/sqlite/sqlite_functions.c

```diff
@@ -416,3 +439,76 @@ void sqlite_library_shutdown(void)
     (void) sqlite3_shutdown();
     spinlock_unlock(&sqlite_spinlock);
 }
+
+int attach_database(sqlite3 *database, const char *db_file, const char *alias, bool in_memory)
+{
+    char buf[1024 + 1] = "";
```

Comment:
This magic number `1024`,  please, can you give a name to it to be more clear the reason you selected it?

Reply:
just a large enough buffer for one line add-hoc commands during initialization